handler/src/test/java/io/netty/handler/ssl/JdkConscryptSslEngineInteropTest.java

```diff
@@ -85,4 +85,10 @@ protected boolean mySetupMutualAuthServerIsValidClientException(Throwable cause)
         // TODO(scott): work around for a JDK issue. The exception should be SSLHandshakeException.
         return super.mySetupMutualAuthServerIsValidClientException(cause) || causedBySSLException(cause);
     }
+
+    @Ignore("Ignore due bug in Conscrypt")
+    @Override
+    public void testHandshakeSession() throws Exception {
```

Comment:
@ejona86 @flooey This also needs to be fixed in conscrypt.

Reply:
@normanmaurer Can you be more specific about what behavior you see in Conscrypt and what you think it should be doing?

Reply:
Basically what this PR fixes. 

session.getLocalCertificates() and getLocalPrincipal() should not return null in the Trustmanager serverside. Check the included unit test

Reply:
Oh, on the server.  Sure, makes sense.  Weird that we don't do that, I've filed https://github.com/google/conscrypt/issues/634 to track fixing it.